.devfile.yaml

```diff
@@ -0,0 +1,126 @@
+# 
+# Copyright 2023 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+schemaVersion: 2.2.0
+metadata:
+  name: devfile-web
+  displayName: Devfile Web Services
+  description: A devworkspace for devfile web services.
+  language: TypeScript
+  projectType: Next.js
+  tags:
+    - TypeScript
+    - Node.js
+    - Next.js
+    - Devfile
+    - Registry
+    - Documentation
+    - devfile.io
+  version: 1.0.0
+components:
+  - name: image-builder
+    image:
+      imageName: "quay.io/{{profile}}/{{project}}:{{imageTag}}"
+      dockerfile:
+        args:
+          - --build-arg=PROJECT_NAME={{project}}
+        uri: Dockerfile
+  - name: runner
+    attributes:
+      container-overrides:
+        securityContext:
+          runAsUser: 1001
+          allowPrivilegeEscalation: false
+          runAsNonRoot: true
+          capabilities:
+            drop: ["ALL"]
+          seccompProfile:
+            type: "RuntimeDefault"
+    container:
+      image: node:16-alpine
```

Comment:
Can the container image be node 18 instead, or does it have to be 16?

Reply:
@mike-hoang I made the runtime image node 16 to match the node version of the runner defined in the [Dockerfile](https://github.com/devfile/devfile-web/blob/0c9f5ecaf52bd726f574240ae730efd8036e6f35/Dockerfile#L81). Should devfile-web be using node 18?